### PR TITLE
Feature/support app version awareness

### PR DESCRIPTION
**Pull request by duartecoelhomovai**

### CODE REVIEW

File: .github/workflows/integration-build-platform.yml [hunk 1]
```diff
@@ -722,7 +722,7 @@ jobs:
 
           # install ros
           sudo apt install -y --allow-downgrades python3-rosnode python3-rosparam python3-rostopic
-          export PYTHONPATH="${PYTHONPATH}:/usr/lib/python3/dist-packages"
+          export PYTHONPATH="${PYTHONPATH}:/usr/lib/python3/dist-packages
```
Comment (diasdm) [inline .github/workflows/integration-build-platform.yml]:
```suggestion
          export PYTHONPATH="${PYTHONPATH}:/usr/lib/python3/dist-packages"
```

File: .github/workflows/multi-py-deb-workflow.yml [hunk 2]
```diff
@@ -270,6 +270,8 @@ jobs:
           git config --global user.name ${{ secrets.auto_commit_user }}
           git config --global user.email ${{ secrets.auto_commit_mail }}
           git add .bumpversion.cfg
+          git add **__version__.py
+          git add setup.py
```
Comment (diasdm) [inline .github/workflows/multi-py-deb-workflow.yml]: Add setup.py?

**Review by diasdm — CHANGES_REQUESTED** [review]

File: .github/workflows/multi-py-deb-workflow.yml (hunk 2 above)
Comment (AlexFernandes-MOVAI) [reply .github/workflows/multi-py-deb-workflow.yml]: setup.py is a link here, should not be committed only the ones in the subdirs

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]
please review the changes in multi-py-deb

**Review by diasdm — APPROVED** [review]

**Pull request merged**